src/nidigital/templates/session.py/fancy_fetch_capture_waveform.py.mako

```diff
@@ -34,15 +34,16 @@
         import collections
         import sys
 
-        data, actual_num_waveforms, actual_samples_per_waveform = self._fetch_capture_waveform(site_list, waveform_name, samples_to_read, timeout)
+        timeout_secs = _converters.convert_timedelta_to_seconds(timeout, _visatype.ViReal64)
+        data, actual_num_waveforms, actual_samples_per_waveform = self._fetch_capture_waveform(site_list, waveform_name, samples_to_read, timeout_secs)
```

Comment:
Why is `_fetch_capture_waveform` not code-generated?
The comment doesn't say so.